tests/transformers/test_transformers_model_export.py

```diff
@@ -2462,3 +2465,140 @@ def test_signature_inference(pipeline_name, data, result, request):
     signature_with_input = mlflow.transformers._get_default_pipeline_signature(pipeline, data)
 
     assert signature_with_input.to_dict() == result
+
+
+@pytest.mark.parametrize(
+    "dtype", [torch.float16, torch.bfloat16, torch.float32, torch.float64, torch.int32, torch.int64]
+)
+@pytest.mark.skipcacheclean
+@pytest.mark.skipif(
+    "Version(transformers.__version__) < Version('4.26.1')", reason="Feature does not exist"
+)
+def test_extraction_of_torch_dtype_from_pipeline(dtype):
+    pipe = transformers.pipeline(
+        task="translation_en_to_fr",
+        model=transformers.T5ForConditionalGeneration.from_pretrained("t5-small"),
+        tokenizer=transformers.T5TokenizerFast.from_pretrained("t5-small", model_max_length=100),
+        framework="pt",
+        torch_dtype=dtype,
+    )
+
+    parsed = mlflow.transformers._extract_torch_dtype_if_set(pipe)
+
+    assert parsed == str(dtype)
+
+
+@pytest.mark.parametrize(
+    "dtype", [torch.float16, torch.bfloat16, torch.float32, torch.float64, torch.int32, torch.int64]
+)
+@pytest.mark.skipcacheclean
+def test_deserialization_of_configuration_torch_dtype_entry(dtype):
+    flavor_config = {"torch_dtype": str(dtype), "framework": "pt"}
+
+    parsed = mlflow.transformers._deserialize_torch_dtype_if_exists(flavor_config)
+    assert isinstance(parsed, torch.dtype)
+    assert parsed == dtype
+
+
+@pytest.mark.skipcacheclean
+@pytest.mark.skipif(
+    "Version(transformers.__version__) < Version('4.26.1')", reason="Feature does not exist"
```

Comment:
```suggestion
    Version(transformers.__version__) < Version('4.26.1'), reason="Feature does not exist"
```

Reply:
ahhhh I thought that the skipif logic needed to be string wrapped. Thanks for pointing this out, Haru! :D